etc/httpd/global.conf

```diff
@@ -126,10 +126,9 @@ RewriteRule ^/ja/?$ http://www.mozilla.jp/ [L,R]
 RewriteRule ^/ja/firefox/organizations/?$ http://www.mozilla.jp/business/downloads/ [L,R=301]
 
 # bug 1138280
-RewriteRule ^/ja/(firefox|thunderbird)/(beta/)?notes/ http://mozilla.jp/$1$2/notes/ [L,R=301]
-RewriteRule ^/ja/mobile/(beta/?)notes/ http://mozilla.jp/android/$1$2/notes/ [L,R=301]
-RewriteRule ^/ja/(firefox|thunderbird)/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozilla.jp/firefox/$1/$2$3/releasenotes/ [L,R=301]
-RewriteRule ^/ja/mobile/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozillla.jp/firefox/android/$1$2/releasenotes/ [L,R=301]
+RewriteRule ^/ja/(firefox|thunderbird)/(beta/)?notes/ http://www.mozilla.jp/$1$2/notes/ [L,R=301]
```

Comment:
Looks to me like this would result in `http://www.mozilla.jp/firefoxbeta/notes/`. That is to say that I think there should be a `/` character between the `$1` and `$2` in the destination URL.